Cython/Compiler/ModuleNode.py

```diff
@@ -3076,7 +3076,9 @@ def generate_module_init_func(self, imported_modules, env, code):
         code.put_setup_refcount_context(header3)
 
         env.use_utility_code(UtilityCode.load("CheckBinaryVersion", "ModuleSetupCode.c"))
+        code.putln('#if !CYTHON_LIMITED_API')
```

Comment:
I think this should be `!CYTHON_COMPILING_IN_LIMITED_API` - that macro is used internally to identify it while the `CYTHON_LIMITED_API` is defined by users to trigger limited api.

It seems a little arbitrary but it's what we've done elsewhere so it's probably best to be consistent